packages/hub/src/graphql/mutations/updateModelSlug.ts

```diff
@@ -45,6 +45,9 @@ builder.mutationField("updateModelSlug", (t) =>
         data: {
           slug: input.newSlug,
         },
+        include: {
+          owner: true,
```

Comment:
what's this for? Right now only the owner can do updates, so I don't understand the need to send that in.

Reply:
Pothos requires it, because it still needs to check authScopes for response; if I omit this, response shape doesn't match GraphQL type.